### ISSUE TITLE
Store existingjobs as attribute rather than reparsing cron file?

### ISSUE THREAD

**Issue by rettigs** [osuosl-cookbooks/rdiff-backup#31]
Doing this would eliminate the error-prone parsing of /etc/cron.d/rdiff-backup in order to get a list of jobs that are already present on the rdiff-backup server.

**Comment by jordane** [comment]
Why do you need to store existing jobs? If a job isn't in chef, it has no place being on the server.

**Comment by rettigs** [comment]
@jordane The reason to store existing jobs was so that during a new Chef run, it could detect which jobs were on the server but *not* in Chef, so that it could remove them; in particular, using the Nagios NRPE LWRP required this in order to call the `:remove` action on that job.  However, as mentioned in https://github.com/osuosl-cookbooks/rdiff-backup/pull/13#issuecomment-106578284, I could just delete the created files manually rather than using the `:remove` action in the LWRP, meaning that the existing jobs would no longer need to be stored.

So basically, this issue should be moot.
